x/ibc-hooks/go.mod

```diff
@@ -3,16 +3,15 @@ module github.com/osmosis-labs/osmosis/x/ibc-hooks
 go 1.21
 
 require (
-	cosmossdk.io/errors v1.0.0
-	github.com/CosmWasm/wasmd v0.40.1
+	cosmossdk.io/errors v1.0.1
+	github.com/CosmWasm/wasmd v0.45.1-0.20231128163306-4b9b61faeaa3
```

Comment:
why do we need to specify a commit here?

Reply:
Automatic `go mod tidy` replaces that happens, note that these are not the actual dep updates happening with us replacing the dep version / commits in the `replace` section below